### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by alcalyn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Fenrir Api client works with either Guzzle3 or Guzzle5.
+
+``` php
+use CanalTP\FenrirApiClient\FenrirApi;
+use CanalTP\FenrirApiClient\AbstractGuzzle\Version;
+
+$baseUrl = 'http://fenrir-api.local/';
+
+$fenrirApi = new FenrirApi(new Version\Guzzle5($baseUrl)); // For Guzzle 5
+$fenrirApi = new FenrirApi(new Version\Guzzle5($baseUrl)); // For Guzzle 3
```
Comment (Netmisa) [inline README.md]: Guzzle5 to Guzzle3
```
$fenrirApi = new FenrirApi(new Version\Guzzle3($baseUrl));
```

**Comment by alcalyn** [comment]
Jenkins test this please

**Comment by alcalyn** [comment]
Jenkins test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Comment by Netmisa** [comment]
Jenkins could you test this please

**Pull request merged**